change/@azure-communication-react-35f5e081-17b0-402a-a4a6-b642d9c2d8b5.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
```

Comment:
Do we need stable changelog item here (as we update only the beta version dependencies)?